ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js

```diff
@@ -251,6 +251,11 @@ export function interactAndWaitForConfigurationManagementScan(interactionCallbac
         routeMatcherMapForConfigurationManagementDashboard
     );
 
+    // ROX-24912
+    // Comment out for now and assume that scan results are available from compliance tests.
+    // Maybe interactAndWaitForResponses waits only for the first pair of responses from page visit.
```

Comment:
I believe this is correct, the single `wait` call in `interactAndWaitForResponses` will only wait for the first response from everything in the `routeMatcherMap` and will continue on after that. Follow up requests/responses from the matcher map will not be waited on.